### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by anuarshaidenov**
## Milestone goals:
- [X] Remove all hardcoded items from the tasks array.
- [X] Create a new JavaScript file for the new functionality.
- [X] Implement a function for adding a new task (add a new element to the array).
- [X] Implement a function for editing task descriptions.
- [X] Implement a function for deleting a task (remove an element from the array).
- [X] Implement a function for the "Clear all completed" button (use filter() method).
- [X] By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- [X] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
- [X] All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/js/listView.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
+/* eslint-disable indent */
+/* eslint-disable class-methods-use-this */
```
Comment (rindrajosia) [inline src/js/listView.js:2]: - [ ] [REQUIRED] Please don't disable eslint. it is like making some changes on the linter files. Keep in mind that linter helps you to have a good codebase quality.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi , 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:

## To highlight :+1: 

- All hardcoded items from the tasks array are removed.
- It has a new JavaScript file for the new functionality.
- User can delete, update and clear all completed tasks.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/listView.js (hunk 1 above)
Comment (anuarshaidenov) [reply src/js/listView.js]: Done✔️ Thanks for the feedback, @rindrajosia

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @anuarshaidenov, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**